Controller/CRUDController.php

```diff
@@ -296,7 +295,6 @@ public function batchActionDelete(ProxyQueryInterface $query)
      *
      * @return Response|RedirectResponse
      *
-     * @throws NotFoundHttpException If the object does not exist
```

Comment:
Why deleting the `@throws NotFoundHttpException`? The exception may be thrown, so it should be documented.